fabric-loot-tables-v1/src/main/java/net/fabricmc/fabric/api/loot/v1/LootEntryTypeRegistry.java

```diff
@@ -30,9 +31,10 @@
 	LootEntryTypeRegistry INSTANCE = LootEntryTypeRegistryImpl.INSTANCE;
 
 	/**
-	 * Registers a loot entry type by its serializer.
+	 * Registers a loot entry type serializer by its ID.
 	 *
+	 * @param id the loot entry's ID
 	 * @param serializer the loot entry serializer
 	 */
-	void register(LootPoolEntry.Serializer<?> serializer);
+	void register(Identifier id, LootPoolEntry.Serializer<?> serializer);
```

Comment:
Because this is a breaking change, you should bump major version in the build.gradle

Reply:
Done